data-common.json

```diff
@@ -25,11 +25,13 @@
   },
   "firefox": {
     "developer": {
+      "versions": ["firefox52", "firefox53"],
```

Comment:
This is data that can be used between various tabs
https://github.com/kangax/compat-table/blob/gh-pages/data-common.json